docs/modules/develop/pages/guide_migrate_webpacker_app.adoc

```diff
@@ -110,29 +110,16 @@ The completely remove Sprockets references from your application:
 * Remove `app/assets` folder
 * In `config/application.rb` replace:
 
-[source,console]
+[source,ruby]
 ----
 require 'rails/all'
 ----
 
 with:
 
-[source,console]
+[source,ruby]
 ----
-require "rails"
-# Pick the frameworks you want:
-require "active_model/railtie"
-require "active_job/railtie"
-require "active_record/railtie"
-require "active_storage/engine"
-require "action_controller/railtie"
-require "action_mailer/railtie"
-require "action_mailbox/engine"
-require "action_text/engine"
-require "action_view/railtie"
-require "action_cable/engine"
-# require "sprockets/railtie"
-require "rails/test_unit/railtie"
+require "decidim/rails"
```

Comment:
💬 Following the idea of the previous comment, here I'd add a comment like # Add the frameworks used by your app that are not loaded by Decidim.

Reply:
Added the comment.